pkg/scheduler/util/utils.go

```diff
@@ -135,8 +135,14 @@ func (l *SortableList) Sort() {
 }
 
 // HigherPriorityPod return true when priority of the first pod is higher than
-// the second one. It takes arguments of the type "interface{}" to be used with
-// SortableList, but expects those arguments to be *v1.Pod.
+// the second one. If two pods' priority are equal, compare their StartTime.
```

Comment:
pods' priority --> pods's priorities

Reply:
In `pods'`, there has already a 's' in the last letter, so the last `s` of `pods's` could be omitted?  `priorities` addressed.